test/integration/services/data/test-get-staging-court-reports.js

```diff
@@ -0,0 +1,62 @@
+const expect = require('chai').expect
+const getStgCourtReports = require('../../../../app/services/data/get-staging-court-reports')
+const CasesSummay = require('wmt-probation-rules').CasesSummary
+const CourtReport = require('wmt-probation-rules').CourtReport
+const OmCourtReports = require('wmt-probation-rules').OmCourtReports
+const helper = require('../../../helpers/data/staging-court-reports-helper')
+
+var inserts = []
+describe('services/data/get-staging-workload', function () {
+  before(function () {
+    return helper.insertDependencies(inserts)
+    .then(function (builtInserts) {
+      inserts = builtInserts
+    })
+  })
+
+  it('should retrive any staging court reports', function () {
```

Comment:
Typo

Also, should retrieve any in the batch?

Reply:
done